charmcraft/commands/store/__init__.py

```diff
@@ -703,6 +726,7 @@ def _get_libs_from_tree(charm_name=None):
         base_dir = pathlib.Path('lib') / 'charms'
         charm_dirs = sorted(base_dir.iterdir()) if base_dir.is_dir() else []
     else:
+        charm_name = create_importable_name(charm_name)
```

Comment:
Use `importable_charm_name` as the result here and in the rest of the function (for example, what it receives is importable_charm_name, too...

Reply:
I've changed the result of this to be `importable_charm_name`, and where it is used to build the directory path. 
The argument to the function I've left as `charm_name`, as theoretically this function could be used with either form, since the result of transforming an already transformed name is the same result.